initializr-web/src/test/java/io/spring/initializr/web/controller/custom/CustomProjectContributor.java

```diff
@@ -44,6 +46,9 @@ public void contribute(Path projectRoot) throws IOException {
 				&& ((CustomProjectDescription) this.description).isCustomFlag()) {
 			Files.createFile(projectRoot.resolve("custom.txt"));
 		}
+
+		diff.ifCUstomFlagChanged((CustomProjectDescription)description, (oldFlag, newFlag) ->
```

Comment:
Again, just temp code to show how it would/could be used in a custom contributor